bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,23 @@
+#Properties file for org.eclipse.passage.ldc.pde.core.templates
+###############################################################################
+# Copyright (c) 2019, 2021 ArSysOp and others
```

Comment:
Only 2021?
There is no _others_ this time.

Reply:
fair point